test/gmp.jl

```diff
@@ -69,6 +69,18 @@ ee = typemax(Int64)
             @test big(typeof(complex(x, x))) == typeof(big(complex(x, x)))
         end
     end
+    @testset "division" begin
+        oz = big(2 // 0)
+        zo = big(0 // 2)
```

Comment:
```suggestion
        oz = big(1 // 0)
        zo = big(0 // 1)
```
The `Rational` constructor will change them to this representation anyway.